easybuild/easyconfigs/g/gencore_variant_detection/gencore_variant_detection-1.0.eb

```diff
@@ -0,0 +1,38 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_Conda'
+
+name = "gencore_variant_detection"
+version = "1.0"
+variant = "Linux-x86_64"
+
+homepage = "https://nyuad-cgsb.github.io/variant_detection/public/index.html"
+description = """ This is a bundled install of many software packages for doing variant detection analysis. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+builddependencies = [('Anaconda3', '4.0.0')]
+
+#Use one of the following  - either an environment.yml file or a remote environment definition
+#sources = ["environment.yml"]
+#environment_file = sources[0]
```

Comment:
Actually, no change is needed at all, the requirement is just that `environment_file` points to the absolute path of the file, otherwise it won't be found.